lightning/src/ln/peer_handler.rs

```diff
@@ -273,6 +275,10 @@ impl ChannelMessageHandler for ErroringMessageHandler {
 		features
 	}
 
+	fn get_genesis_hash(&self) -> ChainHash {
+        ChainHash::using_genesis_block(Network::Bitcoin)
```

Comment:
Oops yeah I meant to figure out what to do here and not hardcode it

Reply:
In this case I assume it would be ok. We could make it return an Option and if we ever None anywhere else for that method we can just propagate that through as None for networks. Basically only this case.